security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,26 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-14"]
+== {cert-manager-operator} 1.14
+
+Issued: 2024-07-03
+
+The following advisory is available for the {cert-manager-operator} 1.14:
+
+* link:https://errata.engineering.redhat.com/advisory/128547[RHEA-2024:128547-02]
+
+[id="cert-manager-operator-new-features-1-14"]
+=== New features and enhancements
+
+*FIPS compliant support*
+
+With this release, FIPS mode is now automatically enabled for {cert-manager-operator}. When installed on an {product-title} cluster in FIPS mode, {cert-manager-operator} ensures compatibility without affecting the cluster's FIPS support status.
+
+*NCM issuer*
+
+The {cert-manager-operator} for {product-title} now supports the NCM issuer. The `ncm-issuer` is a Kubernetes controller that integrates with the Nokia NetGuard Certificate Manager (NCM) PKI system to sign certificate requests. This integration streamlines the process of obtaining non-self-signed certificates for applications, ensuring their validity and keeping them up-to-date.
+
```

Comment:
```suggestion
[id="cert-manager-operator-1-14-0-CVEs"]
=== CVEs
* link:https://access.redhat.com/security/cve/CVE-2023-45288[CVE-2023-45288]

```
@swghosh Feel free to add if anything was missed.

Reply:
```diff
CVE-2023-45288
+ CVE-2024-28180
+ CVE-2020-8559
+ CVE-2024-26147
+ CVE-2024-24783
```